creator-node/src/routes/nodeSync.js

```diff
@@ -308,19 +308,23 @@ async function _nodesync (req, walletPublicKeys, creatorNodeEndpoint, dbOnlySync
           transaction
         })
         const fetchedLatestBlockNumber = fetchedCNodeUser.latestBlockNumber
+        const fetchedLatestClockVal = fetchedCNodeUser.clock
 
         // Delete any previously stored data for cnodeUser in reverse table dependency order (cannot be parallelized).
         if (cnodeUser) {
           // Ensure imported data has higher blocknumber than already stored.
           // TODO - replace this check with a clock check (!!!)
           const latestBlockNumber = cnodeUser.latestBlockNumber
+          const latestClockValue = cnodeUser.clock
 
           if (!dbOnlySync) {
             if ((fetchedLatestBlockNumber === -1 && latestBlockNumber !== -1) ||
-              (fetchedLatestBlockNumber !== -1 && fetchedLatestBlockNumber <= latestBlockNumber)
+              (fetchedLatestBlockNumber !== -1 && (fetchedLatestBlockNumber <= latestBlockNumber && latestClockValue >= fetchedLatestClockVal))
```

Comment:
modify this check from 
`(fetchedLatestBlockNumber <= latestBlockNumber && latestClockValue >= fetchedLatestClockVal)`

to 

`(latestClockValue >= fetchedLatestClockVal)`